blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js

```diff
@@ -2,5 +2,6 @@ import EditFormNewRoute from 'ember-flexberry/routes/edit-form-new';
 
 export default EditFormNewRoute.extend({
   modelProjection: '<%=modelProjection%>',
-  modelName: '<%=modelName%>'
+  modelName: '<%=modelName%>',
+  templateName: '<%=entityName%>',
```

Comment:
I have doubts about using entityName here. May be we shold use formName? But I can mistake. So I just want to make sure we use right property.